11/umbraco-cms/reference/configuration/securitysettings.md

```diff
@@ -48,16 +50,15 @@ At the root level of security you can configure the following
 
 ### Keep user logged in
 
-If set to false a user will be logged out after a specific amount of time has passed with no activity, you can specify this time span in the [global settings](globalsettings.md) with the `TimeOut` key.
+When set to false a user will be logged out after a specific amount of time has passed with no activity. You can specify this time span in the [global settings](globalsettings.md) with the `TimeOut` key.
 
 ### Hide disabled users in backoffice
 
-If this is set to "true" it's not possible to see disabled users, which means it's
-not possible to re-enable their access to the backoffice again. It also means you can't create an identical username if the user was disabled by a mistake.
+When this is set to "true" it's not possible to see disabled users. This means it's not possible to re-enable their access to the backoffice again. It also means you can't create an identical username if the user was disabled by a mistake.
 
 ### Allow password reset
 
-The feature to allow users to reset their passwords if they have forgotten them. By default, this is enabled but if you'd prefer to not allow users to do this it can be disabled at both the UI and API level by setting this value to "false".
+The feature to allow users to reset their passwords if they have forgotten them. By default, this is enabled. It can be disabled at both the UI and API level by setting this value to "false".
```

Comment:
```suggestion
This feature allows users to reset their passwords if they have forgotten them. By default, this is enabled. It can be disabled at both the UI and API level by setting this value to "false".
```